### PR TITLE
feat: load lazily mp3 encoding library for audio recorder

### PR DESCRIPTION
**Pull request by MartinCupela**
### 🎯 Goal

Bundle @breezystack/lamejs separately and thus load it only if audio recording is enabled.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -99,6 +98,7 @@
     "mml-react": "^0.4.7"
   },
   "peerDependencies": {
+    "@breezystack/lamejs": "^1.2.7",
```
Comment (oliverlaz) [inline package.json:101]: I don't think we should make this lib a peer dependency. `await import()` should be enough to instruct the bundler to create a new chunk with it

Comment (MartinCupela) [reply package.json:101]: I will remove it

Comment (oliverlaz) [reply package.json:101]: I mean, it should stay as it was - a regular dependency but bundled in a different chunk

Comment (MartinCupela) [reply package.json]: reverted to regular dependency. I have tested it with devDependency + peerDependency scenario and the resulting app bundle included lamejs in a separate file. That is why I thought devDep would be ok.

**Review by arnautov-anton — APPROVED** [review]

**Pull request merged**